WordPress/Classes/ViewRelated/NUX/WordPressAuthenticator.swift

```diff
@@ -6,6 +6,20 @@ import WordPressUI
 
 
 
+// MARK: - WordPress Endpoint
+//
+public enum WordPressEndpoint {
```

Comment:
Seems like this should be `WordPressCredentials` instead of `WordPressEndpoint`?

Reply:
I've been looking for a proper name for a while. Thanks!! changing right away